[2023-09-23T22:36:22.915+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_test_dag.task_1 manual__2023-09-23T22:36:13.364898+00:00 [queued]>
[2023-09-23T22:36:22.923+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_test_dag.task_1 manual__2023-09-23T22:36:13.364898+00:00 [queued]>
[2023-09-23T22:36:22.923+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-23T22:36:22.941+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): task_1> on 2023-09-23 22:36:13.364898+00:00
[2023-09-23T22:36:22.944+0000] {standard_task_runner.py:57} INFO - Started process 18 to run task
[2023-09-23T22:36:22.947+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'simple_test_dag', 'task_1', 'manual__2023-09-23T22:36:13.364898+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmp4_450r6z']
[2023-09-23T22:36:22.948+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask task_1
[2023-09-23T22:36:22.968+0000] {logging_mixin.py:151} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-23T22:36:23.001+0000] {task_command.py:415} INFO - Running <TaskInstance: simple_test_dag.task_1 manual__2023-09-23T22:36:13.364898+00:00 [running]> on host simple-test-dag-task-1-ntx29avb
[2023-09-23T22:36:23.084+0000] {logging_mixin.py:151} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/providers/cncf/kubernetes/kube_config.py:57 DeprecationWarning: The namespace option in [kubernetes] has been moved to the namespace option in [kubernetes_executor] - the old setting has been used, but please update your config.
[2023-09-23T22:36:23.085+0000] {logging_mixin.py:151} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/providers/cncf/kubernetes/kube_config.py:71 DeprecationWarning: The namespace option in [kubernetes] has been moved to the namespace option in [kubernetes_executor] - the old setting has been used, but please update your config.
[2023-09-23T22:36:23.086+0000] {pod_generator.py:539} WARNING - Model file /opt/airflow/executor-pod-template.yml does not exist
[2023-09-23T22:36:23.131+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='simple_test_dag' AIRFLOW_CTX_TASK_ID='task_1' AIRFLOW_CTX_EXECUTION_DATE='2023-09-23T22:36:13.364898+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-23T22:36:13.364898+00:00'
[2023-09-23T22:36:23.131+0000] {logging_mixin.py:151} INFO - task 1
[2023-09-23T22:36:23.132+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-23T22:36:23.140+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=simple_test_dag, task_id=task_1, execution_date=20230923T223613, start_date=20230923T223622, end_date=20230923T223623
[2023-09-23T22:36:23.199+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-23T22:36:23.231+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
